tests/installation/partitioning/encrypted_full_lvm.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: The test module uses Expert Partitioning wizard on disks with GPT
+# partition table to perform an encrypted full lvm installation using data
+# driven pattern. Data is provided by yaml scheduling file.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use parent 'y2_installbase';
+
+use strict;
+use warnings;
+
+use testapi;
+use version_utils ':VERSION';
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my $test_data = get_test_suite_data();
+
+    my $partitioner = $testapi::distri->get_expert_partitioner();
+    $partitioner->run_expert_partitioner('current');
+
+    my $disk = $test_data->{disks}[0];
+    $partitioner->create_new_partition_table($disk);
+
+    foreach my $partition (@{$disk->{partitions}}) {
+        $partitioner->add_partition_on_gpt_disk({
+                disk      => $disk->{name},
+                partition => $partition
+        });
+    }
+
+    my $volume_group = $test_data->{lvm}->{volume_groups}[0];
```

Comment:
Small thing, but why not to setup all of them in the loop, as our structure allows it? But we can keep it like this for now, no need to change.

Reply:
Something like:
```
    foreach my $vg (@{$test_data->{lvm}->{volume_groups}}) {
        $partitioner->add_volume_group($vg);
        foreach my $lv (@{$vg->{logical_volumes}}) {
            $partitioner->add_logical_volume({
                    volume_group   => $vg->{name},
                    logical_volume => $lv
            });
        }
    }
```

After we merge your PR I would even extract it to common place, as have exactly same piece of code in lvm+RAID1 test.

Reply:
right, I thought about that, it is just the test is thought for one item and as it is in development, it is not guarantee that a loop would work and if the test doesn't need a loop I'd prefer not to read it, just a matter of taste of course, because if we extend that test to use more than one item, I believe that would deserve another test due to the complexity.
Yes, let's keep it for now and if you want to have it different in your PR, I'm glad to hear you point.

Reply:
yes, you are right, ok, let's hide it from the test, so data driven. make sense, if it is not in the test and some shared library.

Reply:
this approach goes in the direction of having 'just one test module' for example, 'partitioning_test_module' which does everything in one line, so from the openQA UI perspective will be more useful to show the test data instead of the code :) but yeah, we are in the good direction, we'll see.

Reply:
I see it differently, we will have modules which would have calls like `setup_lvm`, `setup_raid` and combinations as we need.